### DISCUSSION TITLE
Question: Is there a way to listen to an event when the player controls are showing?

### DISCUSSION THREAD

**Discussion by bernardorubin** [muxinc/elements#650] [Q&A] [answered]
Hi I'm using `MuxPlayer` from `"@mux/mux-player-react"` in `next-js` and want to show a back arrow on top of our full width player: https://kokoro.creatorplus.com/player?id=RQRJiZLB5UP9APpEOzI9V9TAmgnTM5tfTfr3QfEYZQs

I would like the back arrow to show only when the player controls are showing. 
Is there an event listener I can use to achieve showing my button only when the player controls are showing?

**Comment by dylanjha** [comment]
Hi @bernardorubin there's not currently a way to do this. Internally I think `<media-controller>` coming from [Media Chrome](https://github.com/muxinc/media-chrome) has this state, but it's not accessible form the Mux Player level.

Thank you for the suggestion. Sounds like it would be useful to have!

**Comment by luwes** [comment, marked as answer]
I wasn't aware first either but this event actually bubbles up and is a composed event so it passes shadow roots.
it should be as easy as `document.querySelector('mux-player').addEventListener('userinactivechange', callback)`

note that it's currently fires many times, that might be a bug. 

check for the attribute on mediaController by `muxPlayer.mediaController.hasAttribute('user-inactive')`
